doc/rados/operations/monitoring.rst

```diff
@@ -366,12 +378,23 @@ on the number of replicas, clones and snapshots.
 
 - **NAME:** The name of the pool.
 - **ID:** The pool ID.
-- **USED:** The notional amount of data stored in kilobytes, unless the number 
-  appends **M** for megabytes or **G** for gigabytes.
+- **USED:** space allocated for a pool at all OSDs. This includes replication, allocation granularity and erasure coding overhead. Compression savings and object content gaps are taken into account as well. Internally reserved/allocated space (BlueFS part) isn't included.
 - **%USED:** The notional percentage of storage used per pool.
 - **MAX AVAIL:** An estimate of the notional amount of data that can be written
   to this pool.
 - **OBJECTS:** The notional number of objects stored per pool.
+- **STORED:** actual amount of data user/Ceph has stored in a pool. This is similar to the USED column in earlier versions of Ceph but the calculations (for BlueStore!) are more precise (gaps are properly handled).
+- **USED COMPR:** amount of space allocated for compressed data (i.e. this includes comrpessed data plus all the allocation, replication and erasure coding overhead).
+- **UNDER COMPR:** amount of data passed through compression (summed over all replicas) and beneficial enough to be stored in a compressed form. Here is another 'ceph df detail' sample output where 500K object was compressed (partially, due to unaligned object size - 448K only). So 384K at USED COMPR is caused by 3x128K allocation units for keeping compressed data at 3 replicas. And 1.31M at UNDER COMPR column denotes 3x448K blocks of user data compressed at 3 replicas. And this requires in total (see USED) 576K = 3x128K + 3x64K (=ROUND_UP_TO(500K - 448K, 64K))::
+  
+        GLOBAL:
+        SIZE AVAIL USED RAW USED %RAW USED OBJECTS
```

Comment:
this one is also misaligned

Reply:
While we're at it, we might as well explain what "DIRTY" means.

Reply:
it is only meaningful when cache tiering is in use. In that case, it is the number of objects in the cache pool that have been written to the cache pool but not flushed to the base pool yet.

Reply:
The explanation of "DIRTY" has been added to the table.